polling_stations/apps/data_finder/helpers/every_election.py

```diff
@@ -204,3 +204,12 @@ def get_metadata_by_key(self, key):
 
     def get_id_pilot_info(self):
         return self.get_metadata_by_key("2019-05-02-id-pilot")
+
+    @property
+    def multiple_elections(self):
+        if self.has_election and self.request_success:
+            uncancelled_ballots = [b for b in self.ballots if not b["cancelled"]]
+            if len(uncancelled_ballots) > 1:
```

Comment:
Really small thing, but I would return the result of the comparison statement here:
```
return len(uncancelled_ballots) > 1
```
Means you lose the nested if statement and also ensures that the method always explicitly returns either `True` or `False` (currently you implicitly also return `None`)

Reply:
Sounds good - I had originally written it without `if self.has_election and self.request_success:` so there was no implicit `None`. I'd missed it with the extra `if` - that's a sensible way of getting rid of it.